packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.tsx

```diff
@@ -592,9 +592,9 @@ export class DetailsList extends React.Component<IDetailsListProps, IDetailsList
   }
 
   private _onColumnResized(resizingColumn: IColumn, newWidth: number) {
-    this._columnOverrides[resizingColumn.key].calculatedWidth = Math.max(
-      resizingColumn.minWidth || MIN_COLUMN_WIDTH,
-      newWidth);
+    let newCalculatedWidth = Math.max(resizingColumn.minWidth || MIN_COLUMN_WIDTH, newWidth);
+    this.props.onColumnResize(resizingColumn, newCalculatedWidth);
```

Comment:
Hmmm the defaultValue for `onColumnResize` is undefined right? Wouldn't this cause a nullref if you don't pass in the value?